frontend/libs/studio-components/src/components/StudioExpression/SimplifiedEditor/LogicalExpressionEditor/SubExpression/SubExpressionValueSelector/SubExpressionValueReadonly/SubexpressionValueReadonly.tsx

```diff
@@ -44,7 +44,11 @@ const DataModelLookupValue = ({
   return (
     <Binding
       name={texts.readonlyDataModelPath}
-      binding={<StudioCodeFragment>{value.path}</StudioCodeFragment>}
+      binding={
+        <StudioCodeFragment className={classes.datamodelOutputField}>
```

Comment:
What do you think about renaming the className to `ellipsis`? 😊 
```suggestion
        <StudioCodeFragment className={classes.ellipsis}>
```

Reply:
More meaning , I renmed it🙂

Reply:
hmm..🤔 maybe not for this class because we do not aplay `ellipses `in this css , but i can change it in `EditComponentIdRow` component where we use `ellipsis  ` but here maybe we can call it  `overflow `name?